packages/yarnpkg-pnp/package.json

```
@@ -2,8 +2,8 @@
   "name": "@yarnpkg/pnp",
   "version": "2.0.0-rc.6",
   "nextVersion": {
-    "semver": "2.0.0-rc.7",
-    "nonce": "7116580554445175"
+    "semver": "2.0.0",
```

There's a problem here - you scheduled a major (instead of just prerelease) 🙂
```suggestion
    "semver": "2.0.0-rc.7",
```

---

dang - ran `yarn version patch` on each of them instead of that nice `version check -i` Sorry! 😊 . Fix coming up ...

---

fixed